docs/Ipfs-commands.md

```diff
@@ -0,0 +1,37 @@
+The `aragon ipfs` commands can be used to start an IPFS node configured to pin the core Aragon components, inspect hashes, and propagate files to other nodes.
```

Comment:
Do we need to update hack so this change shows up on the page?

Reply:
Yup, will give that a try now, I have to be careful with https://github.com/aragon/hack/issues/116 😅